CMakeLists.txt

```diff
@@ -92,6 +94,12 @@ endif ()
 target_compile_definitions(picowota PUBLIC PICOWOTA_WIFI_SSID=${PICOWOTA_WIFI_SSID})
 target_compile_definitions(picowota PUBLIC PICOWOTA_WIFI_PASS=${PICOWOTA_WIFI_PASS})
 
+# Use the WiFi AP mode upon request
+if (PICOWOTA_WIFI_AP)
```

Comment:
I know it's slightly ugly, but could you add the same "set from environment" bits I have for the SSID/PASS above, just so that setting the build options is consistent?

Something like:
```

if (DEFINED ENV{PICOWOTA_WIFI_AP} AND (NOT PICOWOTA_WIFI_AP))
	set(PICOWOTA_WIFI_AP $ENV{PICOWOTA_WIFI_AP})
	message("Using PICOWOTA_WIFI_AP setting from environment ('${PICOWOTA_WIFI_AP}')")
endif ()
```